versions/3.1.0.md

```diff
@@ -193,11 +193,13 @@ Field Name | Type | Description
 <a name="oasInfo"></a>info | [Info Object](#infoObject) | **REQUIRED**. Provides metadata about the API. The metadata MAY be used by tooling as required.
 <a name="oasServers"></a>servers | [[Server Object](#serverObject)] | An array of Server Objects, which provide connectivity information to a target server. If the `servers` property is not provided, or is an empty array, the default value would be a [Server Object](#serverObject) with a [url](#serverUrl) value of `/`.
 <a name="oasPaths"></a>paths | [Paths Object](#pathsObject) | **REQUIRED**. The available paths and operations for the API.
+<a name="oasWebhooks"></a>webhooks | Map[`string`, [Path Item Object](#pathItemObject)] | The incoming webhooks that MAY be received as part of this API and that the API consumer is expected to implement. Closely related to the `callbacks` feature, this section describes requests initiated other than by an API call, for example by an out of band registration. The key name is a unique string to refer to each webhook, while the Path Item Object describes a request that may be initiated by the API provider and the expected responses. An [example](../examples/v3.1/webhook-example.yaml) is available.
```

Comment:
Just a note that if we are linking examples from the spec, we should ensure links are not broken from the "pretty" ReSpec version on openapis.org.

Reply:
Ah, what's the correct way to refer to the examples? I copied the way the Callback Object did it but @MikeRalphson is correct (of course!) - that link doesn't work in the nicely rendered spec version http://spec.openapis.org/oas/v3.0.2.html#callback-object

Reply:
I'd be happy with either an absolute URL or I could fix relative links in the spec rendering pre-processor. What do people think? It's good that we've spotted that the problem was there already,